README.md

```diff
@@ -83,172 +130,81 @@ Articles, on the other hand, support the following attributes:
   - Keywords
   - Subtitle
   - Summary
-
-## Features
-
-### Caching
-
-The library provides a caching layer to avoid wasting data with very long feeds. All you need to do is provide an expiration date. For more details, give a look to the [Usage section](#usage)
-
-⚠️ For the time being caching is only supported if you are using Kotlin.
-
-### RSS Parsing from string
-
-The library provides a way of simply parsing raw data from a string.
-
-## Installation
-
-The library is uploaded on MavenCentral, so you can easily add the dependency:
-
-```Gradle
-dependencies {
-  implementation 'com.prof18.rssparser:rssparser:<latest-version>'
-}
-```
+  - Season
 
 ## Usage
 
-### Kotlin
+RssParser is available on the Android, iOS and JVM platforms and it's using [Coroutines](https://kotlinlang.org/docs/coroutines-overview.html) for all the asynchronous work.
```

Comment:
```suggestion
RSS Parser is available on Android, iOS, and the JVM. It uses [Coroutines](https://kotlinlang.org/docs/coroutines-overview.html) for all asynchronous work.
```

Reply:
Also, the supported platform list is duplicated here, as you already listed them at the beginning of the README. Consider removing or moving this section, as it's a bit out of place here.